tests/test_packageAlerts.py

```diff
@@ -41,6 +43,15 @@
 import lsst.utils.tests
 import utils_tests
 
+_log = logging.getLogger("lsst." + __name__)
+_log.setLevel(logging.DEBUG)
+
+try:
+    import confluent_kafka  # noqa: F401
+    from confluent_kafka import KafkaException
+except ModuleNotFoundError as e:
+    _log.error('Kafka module not found: {}'.format(e))
```

Comment:
How many of the tests here require kafka? You'll want to use a `skipIf` decorator on the test methods that require kafka, so that they are explicitly skipped. Having a logger here isn't necessary, if you decorate the tests and skip them with an appropriate message.

Reply:
All skipIfs now check if `confluent_kafka = None` which now happens in the imports.